cowprotocol/cowswap
# Improve Appzi NPS/Feedback logging

# Summary

Created new feedback entries for NPS and regular feedback on appzi portal
Adding them as default for test environments to isolate our real user facing answers from tests
![Screen Shot 2022-08-09 at 09 48 41](https://user-images.githubusercontent.com/43217/183606683-534bfaa8-8e10-496d-9fe6-7306d663c97b.png)

# To Test

1. Place a regular feedback. Write the text: `TESTING, IGNORE`
2. Clear the appzi localStorage key, if any
3. Place an order
4. When NPS shows up, fill in the text: `TESTING, IGNORE`
* Open Appzi portal and verify both feedbacks ended in the respective test surveys
